vehicle/psa/identity.go

```diff
@@ -29,21 +51,69 @@ func NewIdentity(log *util.Logger, brand, id, secret string) *Identity {
 			},
 			Scopes: []string{"openid profile"},
 		},
+		brand: brand,
+		realm: realm,
+		vin:   vin,
```

Comment:
Wofür brauchts die VIN? Die Identity hängt doch nur am User, aber nicht mehr am Auto, oder?

Reply:
Eigentlich braucht es die VIN nicht, die hab ich jetzt nur mal verwendet um die neuen refresh_tokens eindeutig in der settings db zu speichern, so dass man den auch anhand der Parameter aus der evcc.yaml wiederfindet. Aber das ist sowas, das noch verbessert werden kann...
Theoretisch sollte sich die Identity bestimmt auch hier auch für mehrere Fahrzeuge nutzen lassen. Ob man da auch ein Subject auslesen kann weis ich gerade aber nicht.

Reply:
Es passiert ja OAuth? Dann wirds auch ein OAuth Token sein und da sollte sich das Subject auslesen lassen. Siehe Tesla.

Reply:
Schaue ich mir mal an wie du das da gelöst hast und ob das bei PSA auch geht

Reply:
Hmm, verstehe ich es richtig dass Tesla hier JWTs zu verwende, die sich dekodieren lassen?
PSA nutzt als Tokens UUIDs (Version 4), die komplett zufällig generiert sind...

Reply:
Mhhm. In dem Fall sollte doch der Username der richtige Key für die Credentials sein, oder?

Reply:
Theoretisch schon, aber ansonsten wird der Username eigentlich nicht gebraucht, weshalb ich den rausgeschmissen hatte.
könnte man natürlich als quasi beliebig definierbaren Account-Identifier wieder reinnehmen

Reply:
Naja- das *ist* der Account Identifier ggü. PSA. Sollte passen?

Reply:
Ja, spricht nichts dagegen. Ich meine nur dass man da im Prinzip dann irgendwas beliebiges eintragen könnte ;)